app/services/migration/migrators/user.rb

```diff
@@ -28,7 +28,7 @@ def ecf_users
 
     def call
       migrate(self.class.ecf_users) do |ecf_user|
-        user = find_or_initialize_user(ecf_user)
+        user = ::Migration::UserMigrator.new(ecf_user).find_initialize_or_merge_user
```

Comment:
Is there a different name we can use? We already have `Migrator::User` which is confusing alongside `Migration::UserMigrator`. What about moving this and `Migration::UserMerger` to namespace them under `Migrator::User` as `Migration::User::Merger` and `Migration::User::Creator` or similar?

Reply:
Renamed to `Migration::Users::Creator` and `Migration::Users::Merger`